src/views/home.hbs

```diff
@@ -1 +1,13 @@
-<h1>Hey!! Welcome</h1>
+<section >
+  
+    <div class="signupSide">
+    <span class="title">إنشاء حساب جديد</span>
+    <form class="signup-form">
+    <input class="input" type="text" name="firstname" placeholder="اسم المستخدم" required autocomplete="off">
+    <input class="input" type="password" name="password" placeholder="كلمة المرور" required autocomplete="off">
+    <input class="input" type="password" name="confirmPassword" placeholder="تأكيد كلمة المرور" required autocomplete="off">
+    <span class="msg">{{msg}}</span>
+    <input type="submit" name="submit" value="أنشئ" class="submit-btn">
+</form>
```

Comment:
@naremanhilles you could tab these elements over based on their heirarchy, like this:
```
<div class="signupSide">
    <span class="title"></span>
    <form class="signup-form">
        <input class="input" ....
    </form>
</div>
```
And so on.  I think this would help to make the code a little more readable.